src/utils/get-usd-value.utils.ts

```diff
@@ -0,0 +1,6 @@
+import {fromNano} from './big-int.utils';
+import {Asset} from '../interfaces/asset.interface';
+
+export const getUsdExchangeRate = (tonPrice: number, asset: Asset) =>
+    (1 / parseFloat(fromNano(BigInt(asset.exchangeRate), asset.decimals))) *
+    tonPrice;
```

Comment:
It's preferable to make a division - the last operation in an equation.  
change `1` with `tonPrice`. U potentially could lose precision on this